Unit3/Module3.15/Mini-project3.15a/README.md

```diff
@@ -0,0 +1,14 @@
+# Mini Project 3.15a: Using Local Storage
```

Comment:
Minor suggestion, we have been including the ShiftUp logo (or logo of other org) to indicate credit for who wrote the assignment. So since you did this one, I suggest you add this Markdown for the logo (and same thing for each of the other READMEs)

##### Credit to Shift_Up coach Raevan Gregson
![Become](https://avatars2.githubusercontent.com/u/38302861?s=200&v=4)